src/gdrive/README.md

```diff
@@ -49,6 +49,21 @@ To authenticate and save credentials:
 
 To integrate this server with the desktop app, add the following to your app's server configuration:
 
+#### Docker
+
```

Comment:
We likely want ot highlight the difference between the npx and the docker version (mostly just passing credentials (which i think in the explizit form is nicer).

Reply:
Ya, we've added a section on the explicit authentication step (which is also a docker cli command). This is a prerequisite step before adding the gdrive server to mcpServers

Reply:
The section is right below this thread:
https://github.com/modelcontextprotocol/servers/pull/368/files#diff-4388c32ec9267085688e3653add6606627eecdac28c0748c171040499695493eR55

Reply:
Great thank you.